actions/user_actions.jsx

```diff
@@ -40,10 +40,15 @@ export async function switchFromLdapToEmail(email, password, token, ldapPassword
     }
 }
 
-export async function loadProfilesAndTeamMembers(page, perPage, teamId = getCurrentTeamId(getState()), success) {
-    const {data} = await UserActions.getProfilesInTeam(teamId, page, perPage)(dispatch, getState);
-    loadTeamMembersForProfilesList(data, teamId, success);
-    dispatch(loadStatusesForProfilesList(data));
+export function loadProfilesAndTeamMembers(page, perPage, teamId = getCurrentTeamId(getState()), success) {
```

Comment:
Can you remove the default argument here? That's using the global state which is going to get confusing.

Reply:
Oh, sorry about this, I thought I removed it.